.github/workflows/devcontainer_make_command.yml

```diff
@@ -150,7 +152,7 @@ jobs:
             CI_CONTAINER_REGISTRY=${{ vars.CI_CONTAINER_REGISTRY }}
             CI_RESOURCE_GROUP=${{ vars.CI_RESOURCE_GROUP }}
             CI_STORAGE_ACCOUNT=${{ vars.CI_STORAGE_ACCOUNT }}
-            ORG_GITHUB_TOKEN=${{ secrets.ORG_GITHUB_TOKEN }}
+            ORG_GITHUB_TOKEN=${{ secrets.FLOWEHR_GITHUB_TOKEN }}
```

Comment:
Have renamed both secret and env var to `FLOWEHR_GITHUB_TOKEN` in my PR

Reply:
cheers – makes sense (didn't do it here because it was just going to make the merge conflicts worse 😅 )